src/components/DroughtThresholds.vue

```diff
@@ -110,24 +125,39 @@ img {
 }
 #title-container {
   position: fixed;
+  background-image: url("./dry_drought_soil_publicDomain.jpg");
```

Comment:
I think this should work if replaced with https://labs.waterdata.usgs.gov/visualizations/images/dry_drought_soil_publicDomain.jpg